loaders/custom-json-loader.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function(source) {
+    const data = JSON.parse(source);
+    const keys = Object.keys(data);
+        keys.forEach((key) => {
```

Comment:
wrong indents

+ it should be chained

+ `reduce` is more suitable for this case